sormas-app/keystore.properties.example

```diff
@@ -1,5 +0,0 @@
-# on linux use / on windows \\ as path seperator
```

Comment:
This example file should not be deleted

Reply:
Deleted by mistake, I will restore the file.

Reply:
done